bash/hail

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I would say who expects to be able to run things in a source distribution without building them?  OK, makes sense.  I would probably use scripts/.